### PR TITLE
Fixed aggregated cells footer styling

### PR DESCRIPTION
**Pull request by alessandrojcm**
The new Mantine TH component makes some styles unnecessary, also fixed the transparent background.

### CODE REVIEW

**Comment by alessandrojcm** [comment]
Not sure why is this trying to merge the upgraded examples commit, I've merged that one to my fork 🤔

File: packages/mantine-react-table/src/table/MRT_Table.tsx [hunk 1]
```diff
@@ -146,7 +146,6 @@ export const MRT_Table = <TData extends Record<string, any> = {}>({
     <Table
       className={clsx(
         'mrt-table',
-        classes.root,
```
Comment (KevinVandy) [inline packages/mantine-react-table/src/table/MRT_Table.tsx:149]: why remove root class?

Comment (alessandrojcm) [reply packages/mantine-react-table/src/table/MRT_Table.tsx:149]: Cause there is no root class in the CSS https://github.com/KevinVandy/mantine-react-table/blob/v2/packages/mantine-react-table/src/table/MRT_Table.module.css

**Comment by KevinVandy** [comment]
> Not sure why is this trying to merge the upgraded examples commit, I've merged that one to my fork 🤔

It's because you named your branch `v2` lol. The github runner runs package upgrades for that branch on push. I recommend a feat- or bugfix- branch name

**Review by KevinVandy — APPROVED** [review]

**Pull request merged**